api_spec.yaml

```diff
@@ -1266,17 +1472,285 @@ paths:
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items:
+                                    oneOf:
+                                        - allOf:
+                                              - $ref: "#/components/schemas/coreConfigType"
+                                              - type: object
+                                                properties:
+                                                    type:
+                                                        type: string
+                                                        enum:
+                                                            - string
+                                                            - boolean
+                                                            - number
+                                                        description: Data type of the property
+                                        - allOf:
+                                              - $ref: "#/components/schemas/coreConfigType"
+                                              - type: object
+                                                properties:
+                                                    type:
+                                                        type: string
+                                                        enum:
+                                                            - enum
+                                                        description: Data type of the property
+                                                    options:
+                                                        type: array
+                                                        items:
+                                                            type: string
+                                                        description: List of possible values for the property in case it is of enum type
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
     /dashboard/api/tenants/login-methods:
         get:
             tags:
                 - Multitenancy
             summary: Get all tenants and available login methods.
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
```

Comment:
tenantId must come from the URL. check this everywhere and ensure it's correct. Only create or update tenant should take tenantId in the body

Reply:
Done